Client/src/Pages/Settings/index.jsx

```diff
@@ -1,9 +1,978 @@
+import ButtonSpinner from "../../Components/ButtonSpinner";
+import Button from "../../Components/Button";
+import "./index.css";
+import AnnouncementsDualButtonWithIcon from "../../Components/Announcements/AnnouncementsDualButtonWithIcon/AnnouncementsDualButtonWithIcon";
+
+const tabList = ["Profile", "Password", "Team"];
+const profileConfig = {
+  firstName: {
+    id: "edit-first-name",
+    label: "First name",
+    type: "input",
+  },
+  lastName: {
+    id: "edit-last-name",
+    label: "Last name",
+    type: "input",
+  },
+  email: {
+    id: "edit-email",
+    label: "Email",
+    description: "After updating, you'll receive a confirmation email.",
+    type: "input",
+  },
+  photo: {
+    id: "edit-photo",
+    label: "Your photo",
+    description: "This photo will be displayed in your profile page.",
+    type: "photo",
+  },
+};
+const passwordConfig = {
+  current_password: {
+    id: "edit-current-password",
+    label: "Current password",
+    type: "input",
+  },
+  password: {
+    id: "edit-password",
+    label: "Password",
+    type: "input",
+  },
+  confirm_password: {
+    id: "notice-confirm-password",
+    label: "Confirm new password",
+    type: "notice",
+  },
+};
+const orgConfig = {
+  name: "Bluewave Labs",
+  isLoading: false,
+  isOpen: false,
+  newName: "",
+};
+const teamColumns = [
+  {
+    id: "checkbox",
+    label: "",
+    sx: { minWidth: "20px", width: "40px" },
+  },
+  {
+    id: "name",
+    label: "NAME",
+    sx: { fontSize: "12px" },
+  },
+  { id: "email", label: "EMAIL", sx: { fontSize: "12px" } },
+  { id: "role", label: "ROLE", sx: { fontSize: "12px" } },
+];
+const teamConfig = [
+  {
+    id: 0,
+    isChecked: false,
+    name: "John Connor",
+    email: "john@domain.com",
+    type: "admin",
+    role: "Administrator",
+    createdAt: "10/4/2022",
+  },
+  {
+    id: 1,
+    isChecked: false,
+    name: "Adam McFadden",
+    email: "adam@domain.com",
+    type: "member",
+    role: "Member",
+    createdAt: "10/4/2022",
+  },
+  {
+    id: 2,
+    isChecked: false,
+    name: "Cris Cross",
+    email: "cris@domain.com",
+    type: "member",
+    role: "Member",
+    createdAt: "10/4/2022",
+  },
+  {
+    id: 3,
+    isChecked: false,
+    name: "Prince",
+    email: "prince@domain.com",
+    type: "member",
+    role: "Member",
+    createdAt: "10/4/2022",
+  },
+];
+const actionsConfig = [
+  {
+    value: "bulk",
+    label: "Bulk actions",
+  },
+];
+const roleConfig = [
+  {
+    value: "role",
+    label: "Change role to",
+  },
+];
 
 const Settings = () => {
-    return (
-        <div>Settings</div>
-    )
-}
+  //(tab) 0 - Profile
+  //(tab) 1 - Password
+  //(tab) 2 - Team
+  const [tab, setTab] = useState("0");
+  const handleTabChange = (event, newTab) => {
+    setTab(newTab);
+  };
+
+  const theme = useTheme();
+
+  const [isLoading, setIsLoading] = useState(false);
+  //TODO - implement delete profile picture function
```

Comment:
Loading state should be handled by the redux store, please see login and register pages for examples.

The relevant redux code is in `Features/auth/authSlice`

Reply:
At the current moment, just for testing purposes I used a single loading state for testing purposes, primarily to have the loading spinners work. Will tackle it in the future once I start tinkering with the redux store.